website/.gitignore

```diff
@@ -0,0 +1,6 @@
+_site
+node_modules
+content/docs/user_guide/compatibility/linux/
+content/.jekyll-cache/
+syscalls.json
+.sass-cache/
```

Comment:
Maybe ignore `work/` too?

Reply:
I've integrated everything with bazel. We actually don't need any explicit ignore options anymore!